### DISCUSSION TITLE
spike/mindstorm hub with powered up remote

### DISCUSSION THREAD

**Discussion by SpudGunMan** [pybricks/support#580] [Show and tell]
using the 3.1 firmware on the new bricks getting remote working is working nicely

```python
from pybricks.hubs import PrimeHub
from pybricks.pupdevices import Remote
from pybricks.parameters import Color, Button
from pybricks.tools import wait

hub = PrimeHub()

print("Connected device now:")
remote = Remote(timeout=5000)
print("Connected device name:", remote.name())

while True:
    pressed = remote.buttons.pressed()
    if Button.RIGHT in pressed:
        print('right')
```

**Comment by RussellM110** [comment]
Hi, how do I get the latest firmware for my Inventor/Prime Hub please?
![image](https://user-images.githubusercontent.com/96078379/148770497-03694d81-0813-4e70-be14-3c9ddbd48289.png)

**Reply by laurensvalk** [reply]
It is not yet released for this hub. See https://github.com/pybricks/support/issues/167

**Reply by RussellM110** [reply]
Hi, Thank you for the message, I installed the 'DualBoot' Firmware last night useing the same link, and the Pybricks interface give me the 'New firmware version' message. is this expected at this time?

**Reply by laurensvalk** [reply]
Correct, it hasn't been updated yet.

But unless you really need the latest version, you can just ignore this message. Just use the old version until we release a new one :)

**Reply by RussellM110** [reply]
ok, many thanks for your support :)
